package.json

```diff
@@ -11,6 +11,9 @@
   "bugs": {
     "url": "https://github.com/dafny-lang/ide-vscode/issues"
   },
+  "activationEvents": [
+    "onLanguage:toml"
```

Comment:
Isn't that a bit too broad? toml is not Dafny specific. This means that the Dafny extension will start even if a toml file is run that isn't a dfyconfig. Could you please create an empty .toml file and verify that it won't crash the language server somehow?

Reply:
Activating too early can not lead to bugs, but it hurts VSCode performance since the extension will trigger startup code even when you're just working with "*.toml" files that are unrelated to Dafny.

I would have preferred to set the extension to activate only when the path pattern "**/*dfyconfig.toml" is opened, but VSCode does not support that.

If I leave out this activation event, then if you open a `dfyconfig.toml` without opening any .dfy file, the extension does not activate. This is only a slight inconvenience, but starting the extension even when you're not doing anything with Dafny is also only slightly inconvenient.

Reply:
Ok, I can live with that for now.